crates/bevy_render/src/render_resource/storage_buffer.rs

```diff
@@ -60,14 +64,19 @@ impl<T: ShaderType + WriteInto> StorageBuffer<T> {
         &mut self.value
     }
 
+    /// Debug label of a buffer. This will show up in graphics debuggers for easy identification.
+    pub fn label(&mut self, label: Option<Cow<'static, str>>) {
```

Comment:
We should probably use a `impl Trait` argument here here to make the ergonomics a bit better.